packages/authentication/src/decorator.ts

```diff
@@ -49,5 +49,7 @@ export function authenticate(strategyName: string, options? : Object) {
  */
 export function getAuthenticateMetadata(controllerObj: Object, methodName: string): IMetadata {
   const metadataObj: AuthenticationMetadata = Reflector.getMetadata('loopback:authenticate', controllerObj, methodName);
-  return metadataObj.getMetadata();
+  if (metadataObj)
```

Comment:
The first argument should be `controllerClass` and we probably should call `Reflector.getMetadata(AUTHENTICATE_KEY, controllerClass.prototype, methodName);`